corehq/apps/reports/standard/monitoring.py

```diff
@@ -1797,15 +1798,30 @@ def _total_row(self, rows, report_data):
             total_row[3] = '%s / %s' % (num, len(rows))
         return total_row
 
+    @property
+    def get_all_rows(self):
```

Comment:
This has two differences from `rows()`:

It doesn't support `view_by_group` - this might be a problem, since I believe the option to export a `view_by_group` version will still be there.  Is it possible to make this only exportable when viewed by user?

It gets users in a different way.  It still ends up assembling a list of users anyways, so I don't think there's any reason not to use this version also for the normal `rows` method - I'd switch `users_to_iterate` this version if it's more performant.